browser-extension/src/utils.js

```diff
@@ -103,3 +103,104 @@ export const getRandomSubarray = (arr, size) => {
 export const getVideoStatistics = (videoId) => {
   return fetchTournesolApi(`videos/?video_id=${videoId}`, 'GET', {});
 };
+
+export const millifyViews = (videoViews) => {
+  videoViews = videoViews.toString().replace(/[^0-9.]/g, '');
+  if (videoViews < 1000) {
```

Comment:
A bit weird to do this number comparison after having converted videoViews to a string. But it is JavaScript, everything is permitted ^^ and it seems to be working!

Reply:
I'll look into it!